packages/nitro-protocol/src/index.ts

```diff
@@ -57,7 +57,7 @@ export {
   randomChannelId,
   randomExternalDestination,
   replaceAddressesAndBigNumberify,
-  setupContracts,
+  setupContract as setupContracts,
```

Comment:
I see that this is part of the exported API of the `nitro-protocol` package.

Q: Who is consuming nitro-protocol's test helpers as a dependency?

Reply:
Good question. I think probably no-one.

I don't think we ought to export it, really.